.gitlab-ci.yml

```diff
@@ -56,18 +56,16 @@ baseline-image:
      - echo $BRANCH
      - export COMMIT=${CI_COMMIT_SHORT_SHA}
      - echo $COMMIT
-     - git clone https://github.com/openebs/e2e-infrastructure.git
+     - git clone https://github.com/$REPO/e2e-infrastructure.git
```

Comment:
@nsathyaseelan why we changed from openebs to $REPO ?
 Is it passed as environment variable in gitlab?